docker-test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -vex
```

Comment:
Why a separate script for this? Why not an option of the existing test script?

Reply:
It is now inside the docker image used for tests.